src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import Image, { ImageProps } from '../Image'
+
+interface NavbarProps {
+  /** Array of JSX elements */
+  links?: JSX.Element[]
+  /** custom style applied to individual links */
+  linkStyle?: string
+  /** Logo can be text or an image */
+  logo: string | HTMLImageElement
+  /** Logo link */
+  logoURL: string
+  // /** Pins navbar (or not) */
```

Comment:
```suggestion
  /** Pins navbar (or not) */
```
We don't need the extra comment tags here

Reply:
will do